### ISSUE TITLE
sahi.utils.shapely module not found error

### ISSUE THREAD

**Issue by ed-muthiah** [obss/sahi#113]
Hey there :)

I'm on a windows 10 machine using conda:

commands to recreate the issue:
```
git clone https://github.com/obss/sahi
cd sahi
conda create -n sahi python=3.6
conda activate sahi
pip install sahi

python
>>> import sahi.utils.coco
````
This error is returned:
```
>>> from sahi.utils import coco                                                                                                                        
Traceback (most recent call last):                                                                                                                       
File "<stdin>", line 1, in <module>                                                                                                                    
File "C:\Users\Annamalai\Documents\sightdata\data\sahi\sahi\utils\coco.py", line 15, in <module>
  from sahi.utils.shapely import ShapelyAnnotation, box, get_shapely_multipolygon                                                                      
File "C:\Users\Annamalai\Documents\sightdata\data\sahi\sahi\utils\shapely.py", line 6, in <module>
  from shapely.geometry import CAP_STYLE, JOIN_STYLE, MultiPolygon, Polygon, box                                                                       File "C:\Users\Annamalai\anaconda3\envs\coco2yolo\lib\site-packages\shapely\geometry\__init__.py", line 4, in <module> 
  from .base import CAP_STYLE, JOIN_STYLE                                                                                                              
File "C:\Users\Annamalai\anaconda3\envs\coco2yolo\lib\site-packages\shapely\geometry\base.py", line 19, in <module>                                      from shapely.coords import CoordinateSequence
File "C:\Users\Annamalai\anaconda3\envs\coco2yolo\lib\site-packages\shapely\coords.py", line 8, in <module>                                              from shapely.geos import lgeos
File "C:\Users\Annamalai\anaconda3\envs\coco2yolo\lib\site-packages\shapely\geos.py", line 154, in <module>                                              _lgeos = CDLL(os.path.join(sys.prefix, 'Library', 'bin', 'geos_c.dll'))
File "C:\Users\Annamalai\anaconda3\envs\coco2yolo\lib\ctypes\__init__.py", line 348, in __init__
  self._handle = _dlopen(self._name, mode)
OSError: [WinError 126] The specified module could not be found
```

**Comment by ed-muthiah** [comment]
This seems to have solved my issue:
`conda install -c conda-forge shapely`

**Comment by fcakyon** [comment]
in windows, shapely needs to be installed by conda as you mentioned. will add it to the readme :)

**Comment by fcakyon** [comment]
This issue will most probably be fixed once the shapely==1.8.0 is released: https://github.com/Toblerity/Shapely/pull/1108

**Comment by ed-muthiah** [comment]
Thanks!
